docs/authoring.md

```diff
@@ -11,6 +11,9 @@ The AsyncAPI generator has been built with extensibility in mind. The package us
 1. Templates may contain "filters" or helper functions. They must be stored in the `.filters` directory under the template directory. [Read more about filters](#filters).
 1. Templates may have a configuration file. It must be stored in the template directory and its name must be `.tp-config.json`. [Read more about the configuration file](#configuration-file).
 1. There are params with special meaning. [Read more about special params](#special-params).
+1. Default parameters that you can use in your templates, except the ones that you defined on your own in the config,  are:
+   - `asyncapi` that is a parsed spec file object. Read the [API](https://github.com/asyncapi/parser-js/blob/master/API.md#AsyncAPIDocument) of the Parser to understand to what structure you have access in this parameter.
+   - `originalAsyncAPI` that is an original spec file before it is parsed. 
 
```

Comment:
```suggestion
1. When using [file templating](https://github.com/asyncapi/generator#file-templating) you have access to the following extra variables:
   - When using `$$channel$$` you have access to two variables `channel` and `channelName`. Where `channel` contains the current channel being rendered.
   - When using `$$message$$` you have access to two variables `message` and `messageName`. Where `message` contains the current message being rendered.
   - When using `$$schema$$` you have access to two variables `schema` and `schemaName`. Where `schema` contains the current schema being rendered.
   - When using `$$parameter$$` you have access to two variables `parameter` and `parameterName`. Where `parameter` contains the current parameter being rendered.
   - When using `$$securityScheme$$` you have access to two variables `securityScheme` and `securitySchemeName`. Where `securityScheme` contains the current security scheme being rendered.
```
Think we should add the file template variables we have access to when using that feature 😄